vips/vip-converter/vip-converter-mainnet/Addresses.ts

```diff
@@ -0,0 +1,54 @@
+export const Assets = [
+  "0xfb6115445Bff7b52FeB98650C87f44907E58f802", // AAVE
+  "0x3EE2200Efb3400fAbB9AacF31297cBdD1d435D47", // ADA
+  "0x8fF795a6F4D97E7887C79beA79aba5cc76444aDf", // BCH
+  "0x250632378E573c6Be1AC2f97Fcdf00515d0Aa91B", // BETH
+  "0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c", // BTCB
+  "0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56", // BUSD
+  "0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82", // CAKE
+  "0x1AF3F329e8BE154074D8769D1FFa4eE058B1DBc3", // DAI
+  "0xbA2aE424d960c26247Dd6c32edC70B295c744C43", // DOGE
+  "0x7083609fCE4d1d8Dc0C979AAb8c869Ea2C873402", // DOT
+  "0x2170Ed0880ac9A755fd29B2688956BD959F933F8", // ETH
+  "0x0D8Ce2A99Bb6e3B7Db580eD848240e4a0F9aE153", // FIL
+  "0xF8A0BF9cF54Bb92F17374d9e9A321E6a111a51bD", // LINK
+  "0x4338665CBB7B2485A8855A139b75D5e34AB0DB94", // LTC
+  "0xCC42724C6683B7E57334c4E856f4c9965ED682bD", // MATIC
+  "0x47BEAd2563dCBf3bF2c9407fEa4dC236fAbA485A", // SXP
+  "0xCE7de646e7208a4Ef112cb6ed5038FA6cC6b12e3", // TRX
+  "0x85EAC5Ac2F758618dFa09bDbe0cf174e7d574D5B", // TRXOLD
+  "0x40af3827F39D0EAcBF4A168f8D4ee67c121D11c9", // TUSD
+  "0x14016E85a25aeb13065688cAFB43044C2ef86784", // TUSDOLD doubt
+  "0xBf5140A22578168FD562DCcF235E5D43A02ce9B1", // UNI
+  "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d", // USDC
+  "0x55d398326f99059fF775485246999027B3197955", // USDT
+  "0xa2E3356610840701BDf5611a53974510Ae27E2e1", // WBETH
+  "0x1D2F0da169ceB9fC7B3144628dB156f3F6c60dBE", // XRP
+  "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63", // XVS
+  "0x0782b6d8c4551B9760e74c0545a9bCD90bdc41E5", // HAY
+  "0x12f31B73D812C6Bb0d735a218c086d44D5fe5f89", // agEUR
+  "0x965F527D9159dCe6288a2219DB51fc6Eef120dD1", // BSW
+  "0x8F0528cE5eF7B51152A59745bEfDD91D97091d2F", // ALPACA
+  "0xf307910A4c7bbc79691fD374889b36d8531B08e3", // ANKR
+  "0x52F24a5e03aee338Da5fd9Df68D2b6FAe1178827", // ANKR Staked BNB
+  "0x4B0F1812e5Df2A09796481Ff14017e6005508003", // TWT
+  "0xCa6d678e74f553f0E59cccC03ae644a3c2c5EE7d", // PLANET
+  "0x12BB890508c125661E03b09EC06E404bc9289040", // RACA
+  "0xfb5B838b6cfEEdC2873aB27866079AC55363D37E", // FLOKI
+  "0x1bdd3Cf7F79cfB8EdbB955f20ad99211551BA275", // BNBX
+  "0xc2E9d07F66A89c44062459A47a0D2Dc038E4fb16", // stkBNB
+  "0xB0b84D294e0C75A6abe60171b70edEb2EFd14A1B", // SnBNB
+  "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c", // WBNB
+  "0x352Cb5E19b12FC216548a2677bD0fce83BaE434B", // BTT
+  "0xaeF0d72a118ce24feE3cD1d43d383897D05B4e99", // WIN
+  "0xd17479997F34dd9156Deef8F95A52D81D265be9c", // USDD
+];
+
+export const BaseAssets = {
```

Comment:
I opted for an array to simplify the iteration process. Additionally, we needed to avoid the baseAsset for each converter, and using an array seemed like a convinient solution for it.

Reply:
Done https://github.com/VenusProtocol/vips/pull/49/commits/9eb73c07ce3c22d0e06dac9f5ba757469d673f1c